src/main/java/seedu/address/logic/commands/EditItemSupplierCommand.java

```diff
@@ -0,0 +1,60 @@
+package seedu.address.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.model.Model.PREDICATE_SHOW_ALL_SUPPLY_ITEMS;
+
+import java.util.Optional;
+
+import seedu.address.commons.core.index.Index;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.item.SupplyItem;
+import seedu.address.model.person.Person;
+
+/**
+ * Edits the details of an existing supply item in the inventory.
+ * Can only be executed through {@code EditCommand}.
+ */
+public class EditItemSupplierCommand extends Command {
```

Comment:
Maybe make this class package-private if you want to prevent accessibility from outside the `commands` package.

Reply:
```suggestion
class EditItemSupplierCommand extends Command {
```

Reply:
Oh, it has to be public due to it being used in unit tests.